cypress/integration/resources.spec.js

```diff
@@ -3,18 +3,36 @@ import {
   RESOURCE_SEARCH,
   RESOURCE_SEARCH_BUTTON,
   RESOURCE_RESET_BUTTON,
+  RESOURCE_TITLE,
   PREV_PAGE_BUTTON,
   NEXT_PAGE_BUTTON,
+  DATA_TEST_CATEGORY,
+  DATA_TEST_LANGUAGES,
+  DATA_TEST_COST,
 } from '../../common/constants/testIDs';
 
 describe('resources', () => {
-  const CostSelect = 'input#react-select-paid-input';
-  const LanguageSelect = 'input#react-select-languages-input';
-  const CategorySelect = 'input#react-select-category-input';
+  const COST_SELECT = 'input#react-select-paid-input';
+  const LANGUAGES_SELECT = 'input#react-select-languages-input';
+  const CATEGORY_SELECT = 'input#react-select-category-input';
+
+  Cypress.Commands.add('checkCustomDataAttribute', (attribute, value) => {
```

Comment:
Can you move this to the custom commands file?

The other one looks specific to this file tho, so it's fine here.

Reply:
Moved it